.gitignore

```diff
@@ -3,5 +3,7 @@ test.db
 .coverage
 .pytest_cache/
 .mypy_cache/
-starlette.egg-info/
+broadcaster.egg-info/
 venv/
+build/
+dist/
```

Comment:
I don't think the `.gitignore` should need to change as part of this pull request?